twisted/test/test_internet.py

```diff
@@ -686,6 +686,7 @@ def test_callLaterUsesReactorSecondsInDelayedCall(self):
             self.assertEqual(call.getTime(), 105)
         finally:
             reactor.seconds = oseconds
+        call.cancel()
```

Comment:
Whoa, how long was this *not* happening??  This seems like a worthwhile patch in and of itself, like a freebie!